bambi/interpret/utils.py

```diff
@@ -1,222 +1,171 @@
-        for the 'slopes' and 'comparisons' effects.
-
-        Depends on the dtype of the variable of interest, effect type, and if
-        self.grid is `True`. The scenarios are described below:
-
-        If numeric dtype and kind is 'comparisons', the returned value is a
-        centered difference based on the mean of `variable'.
-
-        If numeric dtype and kind is 'slopes', the returned value is an epsilon
-        difference based on the mean of 'variable'.
-
-        If categoric dtype the returned value is the unique levels of 'variable'.
-        """
-        values = None  # Otherwise pylint complains
-        terms = get_model_terms(self.model)
-        # get default values for each variable in the model
-        for term in terms.values():
-            if hasattr(term, "components"):
-                for component in term.components:
-                    # if the component is a function call, use the argument names
-                    if isinstance(component, Call):
-                        names = [arg.name for arg in component.call.args]
-                    else:
-                        names = [component.name]
-                    for name in names:
-                        if name == self.name:
-                            predictor_data = self.model.data[name]
-                            dtype = predictor_data.dtype
-                            if component.kind == "numeric":
-                                if self.grid or self.kind == "comparisons":
-                                    predictor_data = np.mean(predictor_data)
-                                if self.kind == "slopes":
-                                    values = self.epsilon_difference(predictor_data, self.eps)
-                                if self.kind == "comparisons":
-                                    values = self.centered_difference(
-                                        predictor_data, self.eps, dtype
-                                    )
-                            elif component.kind == "categoric":
-                                values = np.unique(predictor_data)
-
-        return values
-
-
-@dataclass
-class ConditionalInfo:
-    """
-    Stores information about the conditional (covariates) passed into the
-    'conditional' argument for 'comparisons' and 'slopes' effects
-
-    'ConditionalInfo' is used to create 'slopes' and 'comparisons' data as well
-    as computing the estimates for the 'slopes' and 'comparisons' effects.
+        The fitted Bambi model.
+    target : str
+        Target model parameter (e.g., 'mean', or a distributional component name).
+
+    Returns
+    -------
+    tuple[str, str or None]
+        A tuple containing the response name and the target parameter name.
+        If target is 'mean', returns the response name and the parent parameter.
+        Otherwise, returns the component alias (or response name) and the target (or None).
+    """
+    match target:
+        case "mean":
```

Comment:
I have the recollection that we're going for family-specific parameter names, but I guess we allow for this `"mean"` so we can have it as a default upstream? What about `target` being `str | None` where `None` means whatever is the parent of the family? (I don't love it tbh)